src/components/Footer/Footer.js

```diff
@@ -12,17 +12,29 @@ class Footer extends React.Component {
       <footer className="footer">
         <Container fluid>
           <div className="copyright">
+<<<<<<< Updated upstream
```

Comment:
wha this this Upstream?? something wrong with the way you've pulled. Check once

Reply:
remove those lines, its from the vscode git changes thing

Reply:
I have fixed the code now